libs/expo/betterangels/src/lib/screens/ClientHMIS/tabs/ClientLocationsHmisView/map/hooks/useInteractionsMapRegion.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  TMapDeltaLatLng,
+  coordsToRegion,
+  defaultRegionDelta,
+} from '@monorepo/expo/shared/ui-components';
+import { useMemo } from 'react';
+import { useHmisClientInteractionsMapState } from '../../../../../../state';
+import { HmisNoteQuery } from '../../../../../NotesHmis/HmisProgramNoteEdit/__generated__/hmisGetClientNote.generated';
+
+type TProps = {
+  interaction?: HmisNoteQuery['hmisNote'];
```

Comment:
why are we calling this `interaction` instead of `hmisNote`? Same question for the component names